From e8db11a8f653215ae809f65bc5f9608038d4f70e Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 May 2020 19:14:14 +0300
Subject: [PATCH] [SW-0] make: fix ls error message

The commit: d5349a0 ("[SW-11867] goya2: preboot: pll config changes")
has added Goya2 ARC specific pll.c file, but included it for compilation
on all platforms.
On platforms where is no such file, this resulted in error:
ls: cannot access 'gaudi/pll.c': No such file or directory

Move the addition of the file to compilation targets to Goya2 ARC
specific Makefile.

Change-Id: I4bc104f4d24c0d409b57282b3c76e80b599bb3b7
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index c52af14..3598496 100644
--- a/Makefile
+++ b/Makefile
@@ -28,9 +28,8 @@
 
 include $(HL_DEV)/Makefile
 
-PREBOOT_SRCS_C := $(shell ls common/*.c)
+PREBOOT_SRCS_C += $(shell ls common/*.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV)/common.c)
-PREBOOT_SRCS_C += $(shell ls $(HL_DEV)/pll.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV)/$(HL_TARGET).c)
 PREBOOT_APP_S := $(shell ls app/crt0_preboot.S)
 PREBOOT_APP_C := $(shell ls app/preboot.c)
diff --git a/goya2_arc_gnu/Makefile b/goya2_arc_gnu/Makefile
index eacfb1c..9c22872 100644
--- a/goya2_arc_gnu/Makefile
+++ b/goya2_arc_gnu/Makefile
@@ -1 +1,3 @@
 FW_SIZE := 0x8000
+
+PREBOOT_SRCS_C += $(shell ls $(HL_DEV)/pll.c)
